modules/custom/az_global_footer/data/az_global_footer.json

```diff
@@ -393,14 +393,14 @@
       "link_id": "36",
       "parent_link_id": "0",
       "menu": "az-footer-social-media",
-      "title": "Twitter",
-      "urlpath": "https://twitter.com/uarizona",
+      "title": "X-Twitter",
```

Comment:
Need to verify this title with central Marcomm

Reply:
Has this been verified?

Reply:
Henry Greenberg suggested "X, formerly known as Twitter". Gonna make the change now.